pkg/helm/run.go

```diff
@@ -127,7 +127,7 @@ func Run(flags *hoflags.HelmOperatorFlags) error {
 	// It serves those metrics on "http://metricsHost:operatorMetricsPort".
 	err = kubemetrics.GenerateAndServeCRMetrics(cfg, []string{namespace}, gvks, metricsHost, operatorMetricsPort)
 	if err != nil {
-		log.Info("Could not generate and serve custom resource metrics: ", err.Error())
+		log.Info(fmt.Sprintf("Could not generate and serve custom resource metrics: %s", err.Error()))
```

Comment:
Thanks for this, I also caught this in this PR https://github.com/operator-framework/operator-sdk/pull/1601/files

Do you mind doing that instead, as IMHO it produces a cleaner msg.

Reply:
Ah yes, that looks better :+1: 

I've updated this here to be the same. I'll leave it to you to decide whether to merge this or close it in favour of your one.